code/modules/medical/genetics/bioEffects/hemochromia.dm

```diff
@@ -0,0 +1,278 @@
+ABSTRACT_TYPE(/datum/bioEffect/hemochromia)
+
+/datum/bioEffect/hemochromia_unknown
+	name = "Hemochromia Type-U"
+	desc = "A volatile mutation with 12 known stable alternatives. Will quickly break down into one of them if the subject exits the scanner."
+	id = "hemochromia_unknown"
+	probability = 111
+	effectType = EFFECT_TYPE_MUTANTRACE
+	msgGain = "You feel like you're out of breath."
+	msgLose = "You feel more stable."
+	reclaim_fail = 10
+	lockProb = 1
+	lockedGaps = 2
+	lockedDiff = 2
+	lockedChars = list("A","T","C","G")
+	lockedTries = 12
+	stability_loss = 5
+	icon_state  = "hemochromia_unknown"
+
+	OnLife()
+		if(ishuman(owner))
+			var/mob/living/carbon/human/H = owner
+			if(prob(12))
+				if(!istype(H.loc, /obj/machinery/genetics_scanner))
```

Comment:
```suggestion
		if(ishuman(owner) && prob(12) && !istype(owner.loc, /obj/machinery/genetics_scanner))
			var/mob/living/carbon/human/H = owner
```
(also needs appropriate indentation changes on the next lines)

Reply:
Done.